### PR TITLE
Implemented several docstrings to entity.py

### PR DESCRIPTION
**Pull request by jremes-foss**
Hello. 

Looks like there were several missing docstrings in entity.py. Some of these missing docstrings have been now amended. 

```bin/test quality``` ran locally - all green.

Thank you.

### CODE REVIEW

File: sympy/geometry/entity.py [hunk 1]
```diff
@@ -100,6 +100,7 @@ def __contains__(self, other):
         raise NotImplementedError()
 
     def __getnewargs__(self):
+        """Returns a Tuple that will be passed to __new__ on unpickling."""
```
Comment (smichr) [inline sympy/geometry/entity.py]: Actually, it returns a tuple; Tuple is a SymPy object while tuple is a Python object.

Comment (Newman101) [inline sympy/geometry/entity.py]: Thanks for the comment. I will amend this. :+1:

**Comment by smichr** [comment]
It's in -- thanks!

**Comment by Newman101** [comment]
You're welcome, happy to help. :)

**Pull request merged**